sql/analyzer/parallelize.go

```diff
@@ -22,6 +22,10 @@ func shouldParallelize(node sql.Node, scope *Scope) bool {
 	}
 
 	// Do not try to parallelize DDL or descriptive operations
+	return !isDdlNode(node)
+}
+
+func isDdlNode(node sql.Node) bool {
```

Comment:
I was on the fence about this, but decided not to add another identifier-only interface for this. Especially since my definition of "DDL" here includes things like various SHOW statements. I added a comment though.